base-helm-configs/glance/glance-helm-overrides.yaml

```diff
@@ -118,13 +118,7 @@ conf:
     uwsgi:
       processes: 2
       threads: 1
-  policy:
```

Comment:
I think this is one we should keep. Our standard deployment assumes that you will create a glance_admin role, which is a fairly easy thing to adopt as a deployment practice while also creating a more secure / regulated image policy by default.